palantir-java-format-native/build.gradle

```diff
@@ -69,12 +68,6 @@ graalvmNative {
     }
 }
 
-
-// The image requires parameters to be passed to; eg. gw nativeRun -Pargs=--palantir,-i,<file>
-tasks.named("nativeRun", org.graalvm.buildtools.gradle.tasks.NativeRunTask) {
```

Comment:
I don't think it makes sense to keep this here - I usually just run the native image directly